Sources/Core/Filters/Root/RootFilterViewController.swift

```diff
@@ -39,7 +39,12 @@ final class RootFilterViewController: FilterViewController {
     }()
 
     private var searchFilter: Filter? {
-        return filter.subfilters.first { $0.key == config.searchFilterKey }
+        return filter.subfilters.first { $0.key == config.searchFilter }
+    }
+
+    private func predicateForMutuallyExclusiveFilters(excluding filter: Filter) -> ((Filter) -> Bool) {
```

Comment:
This function isn't used anywhere

Reply:
Nice catch, it's a leftover from my experiments 😅